docker/README.md

```diff
@@ -38,3 +38,42 @@ following GitHub repositories:
 *   https://github.com/GoogleCloudPlatform/redis-docker
 *   https://github.com/GoogleCloudPlatform/sonarqube-docker
 *   https://github.com/GoogleCloudPlatform/wordpress-docker
+
+## Functional tests
+
+For more information on how we test the Docker images, see
+[Docker Container Functional Tests](https://github.com/GoogleCloudPlatform/runtimes-common/tree/master/functional_tests).
+
+## Generate Dockerfile from template
+
+We use [`dockerfile`](https://github.com/GoogleCloudPlatform/runtimes-common/tree/master/versioning)
+tool to generate versionsed `Dockerfiles` from on a common template.
+
+## Generate Cloud Build configuration
+
+We use [`cloudbuild`](https://github.com/GoogleCloudPlatform/runtimes-common/tree/master/versioning)
+tool to generate a configuration file to build `Dockerfiles` using
+[Google Cloud Build](https://cloud.google.com/container-builder/docs/).
+
```

Comment:
We are building container images, not `Dockerfiles`.

Reply:
Thanks, done